pkg/platform/platform.go

```diff
@@ -140,7 +140,7 @@ type Platform interface {
 	// Get resources used by the cloudlet
 	GetCloudletInfraResources(ctx context.Context) (*edgeproto.InfraResourcesSnapshot, error)
 	// Get cluster additional resources used by the vms specific to the platform
-	GetClusterAdditionalResources(ctx context.Context, cloudlet *edgeproto.Cloudlet, vmResources []edgeproto.VMResource, infraResMap map[string]edgeproto.InfraResource) map[string]edgeproto.InfraResource
+	GetClusterAdditionalResources(ctx context.Context, cloudlet *edgeproto.Cloudlet, vmResources []edgeproto.VMResource, deprecatedInfraResMap map[string]edgeproto.InfraResource) map[string]edgeproto.InfraResource
```

Comment:
Why keep `deprecatedInfraResMap`?

Reply:
I think I was originally planning to avoid changing all the implementors, but then I realized I wanted to make sure it wouldn't be used in the future accidentally. So yeah, I don't think there's any reason to keep it. I'll remove it.